signalfx-org-metrics/metrics.yaml

```diff
@@ -1,143 +1,267 @@
+  title: sf.org.apm.grossSpanBytesReceived
 
-sf.org.apm.grossSpansReceivedByToken:
-  brief: The number of spans SignalFx receives for a specific access token
-    before filtering and throttling.
-  description: 'The number of spans SignalFx receives for a specific access token
-    before filtering or throttling.
+sf.org.apm.numSpanBytesReceived:
+  brief: The number of bytes Splunk APM accepts for a span after decompression
+    after filtering and throttling.
+  description: 'The number of bytes Splunk APM accepts for a span after
+    decompression after filtering and throttling.
 
-    Dimension(s): `orgId`, `tokenId`
+    Dimension(s): `orgId`
 
     Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.grossSpansReceivedByToken
+  title: sf.org.apm.numSpanBytesReceived
 
-sf.org.apm.numSpansReceived:
-  brief: The number of spans SignalFx accepts after filtering and throttling.
-  description: 'The number of spans SignalFx accepts after filtering and throttling.
+sf.org.apm.grossSpanBytesReceivedByToken:
+  brief: The number of bytes Splunk APM receives for a specific access token
+    from spans after decompression but before filtering and throttling.
+  description: 'The number of bytes Splunk APM receives for a specific access token
+    from spans after decompression but before filtering and throttling.
 
-  Dimension(s): `orgId`
+  Dimension(s): `orgId`, `tokenId`
 
   Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.numSpansReceived
+  title: sf.org.apm.grossSpanBytesReceivedByToken
 
-sf.org.apm.numSpansReceivedByToken:
-  brief: The number of spans SignalFx accepts for a specific access token
-    after filtering and throttling.
-  description: 'The number of spans SignalFx accepts for a specific access token
-    after filtering and throttling.
+sf.org.apm.numSpanBytesReceivedByToken:
+  brief: The number of bytes Splunk APM accepts for a specific access token
+    for a span after decompression after filtering and throttling.
+  description: |
+    The number of bytes Splunk APM accepts for a specific access token
+    for a span after decompression after filtering and throttling.
 
-  Dimension(s): `orgId`, `tokenId`
+    Dimension(s): `orgId`, `tokenId`
 
-  Data resolution: 1 second'
+    Data resolution: 1 second
   metric_type: counter
-  title: sf.org.apm.numSpansReceivedByToken
+  title: sf.org.apm.numSpanBytesReceivedByToken
 
-sf.org.apm.grossSpanBytesReceived:
-  brief: The number of bytes SignalFx receives from spans after decompression
-    but before filtering and throttling.
-  description: 'The number of bytes SignalFx receives from spans after
-    decompression but before filtering and throttling.
+sf.org.apm.numSpansDroppedInvalid:
+  brief: The number of invalid spans Splunk APM receives.
+  description: 'The number of invalid spans Splunk APM receives. A span can be invalid
+    if there is no start time, trace ID, or service associated with it.
```

Comment:
too many spans in trace is another reason we would increment invalid spans dropped.